### PR TITLE
BaSP-TG-37: list and delete time sheet

### PR DESCRIPTION
**Pull request by Licamus**
I tried to reutilize the CSS shared by @camilavaleriarojas in the slack group to have a uniform style in the app.

### CODE REVIEW

File: src/Components/TimeSheets/index.js [hunk 1]
```diff
@@ -1,11 +1,71 @@
-import styles from './time-sheets.module.css';
+import { useEffect, useState } from 'react';
+import styles from './list.module.css';
+import TimeSheet from './TimeSheet/index';
+import Modal from './Modal/index';
+
+const TimeSheets = () => {
+  const [timeSheets, setTimeSheet] = useState([]);
+  const [showModal, setShowModal] = useState(false);
+  const [timeSheetId, setTimeSheetId] = useState(undefined);
```
Comment (lmalvasia) [inline src/Components/TimeSheets/index.js:9]:
```suggestion
  const [timeSheetId, setTimeSheetId] = useState();
```

File: src/Components/TimeSheets/TimeSheet/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Trash from '../../../assets/trash.png';
```
Comment (lmalvasia) [inline src/Components/TimeSheets/TimeSheet/index.js:1]: Add a new line here to separate the imports from the const.

File: src/Components/TimeSheets/TimeSheet/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          alt="trash"
+          onClick={() => {
+            setShowModal(true);
+            setTimeSheetId(timeSheet._id);
+          }}
+        />
+      </td>
+    </tr>
+  );
+};
```
Comment (lmalvasia) [inline src/Components/TimeSheets/TimeSheet/index.js:25]: New line here.

File: src/Components/TimeSheets/TimeSheet/index.js (hunk 2 above)
Comment (Licamus) [reply src/Components/TimeSheets/TimeSheet/index.js:1]: done!

File: src/Components/TimeSheets/index.js (hunk 1 above)
Comment (Licamus) [reply src/Components/TimeSheets/index.js]: done!

File: src/Components/TimeSheets/TimeSheet/index.js (hunk 3 above)
Comment (Licamus) [reply src/Components/TimeSheets/TimeSheet/index.js:26]: done!

**Pull request merged**